frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -19,6 +21,8 @@
 public class WorkerLifeCycle {
 
     private static final Logger logger = LoggerFactory.getLogger(WorkerLifeCycle.class);
+    private static final Pattern PID_LOG_PATTERN =
+            Pattern.compile(".*\\[PID\\](\\d+)$");
```

Comment:
Please avoid regex as possible as you can.

Reply:
Same response as above